HTMessaging/Classes/Connection.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Connection.swift
+//  HTMessaging
+//
+//  Created by Hector Jose Alvarado Chapa on 22/12/19.
+//  Copyright © 2019 Hector Jose Alvarado Chapa. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+public final class Connection {
+    func withFirebaseFile(with fileName: String, ofType: Filetype, completion: @escaping(_ status: FIRResponse?) -> Void) {
```

Comment:
What if we make these methods static?  Remember to use SendBird as a reference.
I am not sure what we would do with this Connection object. If i'm ot understanding correctly we can discuss this of course